README.md

```diff
@@ -113,6 +113,7 @@ http {
                 Expiration leeway for access_token renewal.
                 If this is set, renewal will happen access_token_expires_leeway seconds before the token expiration.
                 This avoids errors in case the access_token just expires when arriving to the OAuth Resoource Server.
+             --force_reauthorize = true
```

Comment:
it's an example that is commented out; so typically when including it you'd want it to be "true"; but I agree, it is not very consistent with the other options...